aiohttp/web_runner.py

```diff
@@ -128,6 +129,29 @@ def name(self) -> str:
             ssl=self._ssl_context, backlog=self._backlog)
 
 
+class NamedPipeSite(BaseSite):
+    __slots__ = ('_path', )
+
+    def __init__(self, runner: 'BaseRunner', path: str, *,
+                 shutdown_timeout: float=60.0) -> None:
+        super().__init__(runner, shutdown_timeout=shutdown_timeout)
+        self._path = path
+
+    @property
+    def name(self) -> str:
+        return self._path
+
+    async def start(self) -> None:
+        await super().start()
+        loop = asyncio.get_event_loop()
+        server = self._runner.server
+        assert server is not None
+        _server = await loop.start_serving_pipe(  # type: ignore
```

Comment:
Why ignore the type?

Reply:
`start_serving_pipe` is not defined in AbstractEventLoop. So it throws the following
```
aiohttp\connector.py:1149: error: Invalid type "asyncio.ProactorEventLoop"
aiohttp\connector.py:1166: error: "AbstractEventLoop" has no attribute "create_pipe_connection"; ma
ybe "create_connection" or "create_unix_connection"?
```
Is there some other way I'm missing?

Reply:
Dunno. Maybe @asvetlov knows a better way